scripts/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+if [ -z $1"" ]; then
+  echo 'Please add commit hash to be deployed'
+  echo ''
+  exit
+fi
+
+mkdir -p /cb-connect/logs/
+
+DIRECTORY=/cb-connect/bundles/$1
+
+if [ ! -d "$DIRECTORY" ]; then
+  echo "Bundle not found for commit $1"
+  exit
+fi
+
+
+CONFIG=$DIRECTORY/settings.json
+/opt/nodejs/bin/forever stop cb-connect
+
+cd $DIRECTORY
+
+export COMMIT_HASH=$1
+export ROOT_URL='https://connect.codebuddies.org'
+export MONGO_URL='mongodb+srv://rootuser:ns2HcknGze79V9N3@prod-e8gy0.mongodb.net/cb-connect'
```

Comment:
Instead of hardcoding it here, can we set it up as an environment variable manually in DO? I don't think this needs to be part of the script.

Reply:
I know @lpatmo mentioned it's no longer valid, but from my understanding, every time we deploy does this script gets ran and updates the variables in the droplet?

Reply:
Right now, deployment is manual -- we have to ssh into the droplet and run ./deploy. Two todos we have are:
1/ Move the mongourl creds into settings.json
2/ Put the deployment script into a webhook so that changes to the `master` branch trigger an automatic deployment

Reply:
Right, so to my understanding, every time we run `./deploy`, then the environment variables get reset, and if the `mongo_url` is no longer valid, then we're setting the variables with incorrect values. Am I assuming that correctly? 

Instead of using settings.json, would now be a good time to bring up #32 again? I think that would make things more straightforward.

Reply:
#32 is about using .dotenv, not a production settings.json file living in the droplet?

Reply:
Correct - we'd drop settings.json all together and use `.env` files instead. Our local `.env` files will be in ignored with git (we'll use `.env.example` as a way for contributors to see what variables they'd need to get) and define process variables manually in DO.

Reply:
Ah, cool! Sure, that sounds like a good alternative.